lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -521,9 +522,8 @@ def __pub_set_param(self, param, value, check_mode=False):
         Returns:
             True if successful, False otherwise.
         """
-        query = ("ALTER PUBLICATION %s SET (%s = '%s')" % (pg_quote_identifier(self.name, 'publication'),
-                                                           param, value))
-        return self.__exec_sql(query, check_mode=check_mode)
+        query = ("ALTER PUBLICATION %s SET (%s = " % (pg_quote_identifier(self.name, 'publication'), param))
+        return self.__exec_sql(query + '%(val)s)', query_param={'value': value}, check_mode=check_mode)
```

Comment:
```suggestion
        return self.__exec_sql(query + '%(value)s)', query_param={'value': value}, check_mode=check_mode)
```
Could it be that this line is not covered by integration tests?

Reply:
when i finish this i should get a pause in participation. definitly